### DISCUSSION TITLE
UI - Media Manager improvements

### DISCUSSION THREAD

**Discussion by sethdaily** [tinacms/tinacms#4476] [Ideas]
Adding, editing, replacing, and deleting images can be improved.

Solution:

- Support co-location of media
- Improve Media Manager UI
- Fix delete bug
- Add AI for alt text.

![image](https://github.com/tinacms/tinacms/assets/130118701/560d6317-db22-4d12-8002-59a32124fbd4)
**Figure: The media manager UI**

**Comment by drwharris** [comment]
You only get 1 first impression. Anything that improves the UX should be prioritised.

**Comment by Calinator444** [comment]
I think the alt text generator is a great idea. Users should at least be given a default alt description for an image after uploading a file. You should never have an image on your site without alt text. It's not just important for SEO, it also improves accessibility by giving screen readers something to announce for the user.

**Reply by rexolion** [reply]
Just 'Something to announce' can be overwhelming and not informative for screen reader users. It would be especially redundant to announce decorative images with no informational value.

**Comment by architchandra** [comment]
I wish there were a match/ignore paths config for the media folder as well. `.keep` and `.DS_Store` files are showing up in the media manager for me.

**Comment by andrictham** [comment]
I'd love to have the ability to add metadata to media assets using a custom schema. If that's not possible, a basic tagging system would suffice. 

Being able to filter images by those that are used in the content currently being edited would be neat too, so it's possible to replace media quickly

**Comment by obfuscatedgenerated** [comment]
Was asked to write this here (originally #6085)

**Support an "accept" value on a per image field basis for filtering file types in media selection**

I provide a block to render pdfs, and I use the "image" field type to allow the editor to select or upload a file. It would be beneficial to filter the media selector that pops up to only show files with a certain extension, and to only allow uploads of the correct type onto that field.
I currently use a validator to ensure the correctness (`validate: (value) => value && (value.endsWith(".pdf") || "The file must be a .pdf")`), but it would be a lot better if the media selector that pops up only shows the right files in the first place.
